### PR TITLE
Add note regarding loading open-sans from mozcdn

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,18 @@ aware of.
...
 - Tabzilla no longer needs any fonts bundled with it so, we are no longer including
-`@font-face` declarations for Open Sans in the CSS.
+`@font-face` declarations for Open Sans in the CSS. *NOTE:* You can still load
+the Open-Sans fonts using the Mozilla CDN, or another CDN of your choosing, for example:
+
+```CSS
+@font-face{
+	font-family:'Open Sans Light';
+	src:url(//mozorg.cdn.mozilla.net/media/fonts/OpenSans-LightItalic-webfont.eot);
+    font-weight:400;
```
Comment (alexgibson) [inline README.md]: Nit: fix indentation

File: README.md [hunk 2]
```diff
@@ -11,7 +11,18 @@ aware of.
...
 website using one of the options below.
 - Tabzilla no longer needs any fonts bundled with it so, we are no longer including
-`@font-face` declarations for Open Sans in the CSS.
+`@font-face` declarations for Open Sans in the CSS. *NOTE:* You can still load
+the Open-Sans fonts using the Mozilla CDN, or another CDN of your choosing, for example:
+
+```CSS
+@font-face{
+	font-family:'Open Sans Light';
+	src:url(//mozorg.cdn.mozilla.net/media/fonts/OpenSans-LightItalic-webfont.eot);
```
Comment (alexgibson) [inline README.md]: Any reason not to always use `https://` here? Also, maybe `.woff` is a better example?

File: README.md [hunk 3]
```diff
@@ -11,7 +11,18 @@ aware of.
 - Tabzilla is no longer available as a hosted service. You have to add it to your
 website using one of the options below.
 - Tabzilla no longer needs any fonts bundled with it so, we are no longer including
-`@font-face` declarations for Open Sans in the CSS.
+`@font-face` declarations for Open Sans in the CSS. *NOTE:* You can still load
+the Open-Sans fonts using the Mozilla CDN, or another CDN of your choosing, for example:
+
+```CSS
+@font-face{
+	font-family:'Open Sans Light';
```
Comment (alexgibson) [inline README.md]: Teeny nit: space after `:` here and below

File: README.md [hunk 4]
```diff
@@ -11,7 +11,18 @@ aware of.
...
+	src:url(//mozorg.cdn.mozilla.net/media/fonts/OpenSans-LightItalic-webfont.eot);
+    font-weight:400;
+	font-style:italic
```
Comment (alexgibson) [inline README.md]: Nit: missing `;`

**Comment by alexgibson** [comment]
Few things to tidy up, but otherwise looks good :+1: 

The only other thing that springs to mind is should we include a full example of loading a font face declaration, which includes more than one file format?

File: README.md (hunk 2 above)
Comment (schalkneethling) [inline README.md]: No reason, I copied this from MDN :)

**Comment by schalkneethling** [comment]
I was on the fence on that one. Thought it might be a little overkill but, seeing you are thinking it too, let me add a full example.

**Comment by alexgibson** [comment]
r+ :dollar:

**Pull request merged**